OmsAgent/extension-test/omsfiles/oms_extension_run_script.py

```diff
@@ -80,13 +80,25 @@ def is_vm_supported_for_extension():
                        'oracle' : ['6', '7'], # Oracle
                        'debian' : ['8', '9'], # Debian
                        'ubuntu' : ['14.04', '16.04', '18.04'], # Ubuntu
-                       'suse' : ['12']} #SLES
+                       'suse' : ['12'], 'sles' : ['15'] # SLES
 
     try:
         vm_dist, vm_ver, vm_id = platform.linux_distribution()
     except AttributeError:
         vm_dist, vm_ver, vm_id = platform.dist()
 
+    if not vm_dist and not vm_ver: # SLES 15
+        with open('/etc/os-release', 'r') as fp:
+            for line in fp:
+                if line.startswith('ID='):
+                    vm_dist = line.split('=')[1]
+                    vm_dist = vm_dist.split('-')[0]
+                    vm_dist = vm_dist.replace('\"', '').replace('\n', '')
+                elif line.startswith('VERSION_ID='):
+                    vm_ver = line.split('=')[1]
+                    vm_ver = vm_ver.split('.')[0]
+                    vm_ver = vm_ver.replace('\"', '').replace('\n', '')
+
```

Comment:
Why do we need to do this specifically for SLES 15?

Reply:
`platform.dist()` / `platform.linux_distribution()` returns empty strings on SLES 15 :|